Tools/TestWebKitAPI/Tests/WebCore/ImageBufferTests.cpp

```diff
@@ -44,7 +45,7 @@ static ::testing::AssertionResult imageBufferPixelIs(Color expected, ImageBuffer
     auto got = Color { SRGBA<uint8_t> { frontPixelBuffer->item(0), frontPixelBuffer->item(1), frontPixelBuffer->item(2), frontPixelBuffer->item(3) } };
     if (got != expected) {
         // Use this to debug the contents in the browser.
-        // WTFLogAlways("%s", imageBuffer.toDataURL("image/png"_s).latin1().data());
+        WTFLogAlways("%s", imageBuffer.toDataURL("image/png"_s).latin1().data());
```

Comment:
Did you uncomment this statement unintentionally?

Reply:
Yes, reverted, thansk!